runtime/doc/eval.txt

```diff
@@ -3435,11 +3435,11 @@ getchar([expr])						*getchar()*
 			:endfunction
 <
 		You may also receive synthetic characters, such as
-		|<CursorHold>|. Often you will want to ignore this and get
+		|<LeftMouse>|. Often you will want to ignore this and get
```

Comment:
`<CursorHold>` pseudokey doesn't exist any more (replaced by K_EVENT).

Reply:
`<LeftMouse>` is user input, just like `<F10>` or any other special key without canonical unicode representation, of which there are many. Seems weird to ignore only it specifically.  CursorHold needed extra treatment here because it was generated by nvim without user intervention so `getchar()` terminated early, other special chars do not have this problem.

Reply:
Good point. Maybe we can remove this blurb from the docs?

Reply:
I think so. The concrete examples above of how to actually _handle_ pseudokeys (including mouse) should be enough.